packages/ui/Button.tsx

```diff
@@ -130,7 +132,9 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
           </svg>
         </div>
       )}
-      {EndIcon && <EndIcon className="-mr-1 inline h-5 w-5 ltr:ml-2 rtl:mr-2" />}
+      {EndIcon && (
+        <EndIcon className={classNames("-mr-1 inline h-5 w-5 ltr:ml-2 rtl:mr-2", endIconClassName || "")} />
```

Comment:
LGTM :) NIT this could follow the standard we have been using with 

`endIconClassName && endIconClassName`

Reply:
@agustif do you wanna add this or nah?

alternatively, @sean-brydon can you use the "suggest" feature and offer a solution here?